### PR TITLE
Fixes #216

### LINKED ISSUE #216: Rename Annotation -> PropertyAnnotation

### PR DESCRIPTION
**Pull request by obuchtala**

### CODE REVIEW

**Comment by obuchtala** [comment]
@michael FYI

File: model/Annotation.js [hunk 1]
```diff
@@ -1,85 +1,2 @@
...
-  },
-  endPath: {
-    get: function() {
-      return this.path;
-    }
-  }
-});
-
-module.exports = Annotation;
+console.error('DEPRICATED: Use require("substance/model/PropertyAnnotation") instead.');
```
Comment (michael) [inline model/Annotation.js]: It's "deprecated" :)

**Comment by michael** [comment]
Like the consistent naming!

**Comment by michael** [comment]
Will test after my tocpanel iteration.

**Comment by obuchtala** [comment]
Should be pretty harmless, as I have introduced a delegator with the old name.

**Pull request merged**